data/human/human missions.txt

```diff
@@ -5428,3 +5428,66 @@ mission "Deal Change 2"
 				die
 			label skip
 			`	After a few minutes, the pirates finish unloading the cargo, and Ci steps forward, holding out a credit chip. "<payment>, for making the right choice." Ci and the pirates return to their ships as you take off and head towards the main spaceport of <planet>.`
+
+
+
+government "Escort (Betraying)"
+	"display name" "Merchant"
+	swizzle 5
+	
+	"player reputation" 10
+	"attitude toward"
+		"Korath" -.01
+	"fine" 0
+	"friendly hail" "friendly civilian"
+	"hostile hail" "hostile pirate"
+
+mission "Escort offer (Betrayal)"
+	landing
+	name "Escort to <planet>"
+	description "The captain of the <npc> has offered to fly alongside you, protecting you from pirates, on your way to <destination>. In exchange, he has asked for 80,000 credits upon your arrival."
+	source Sunracer
+	destination Maelstrom
+	to offer
+		"armament deterrence" <= 2
+		"cargo attractiveness" >= 2.8
+	on offer
+		conversation
+			`Almost immediately after you land, a heavily modified Firebird with oversized guns follows you down, landing on the pad next to you. A young man with short hair steps out, walks over to your ship, and says, "Hey, you've got a lot of cargo space there. I bet you make loads of dough with it, trading or running jobs. Say, luxury goods are quite cheap here, aren't they? I reckon you'd make an absolute shedload running 'em from here to <planet>. At least 200k, I'll bet. It's in <system>, if you're not familiar with it. Lot of pirates round there, though. I can fight 'em, but there's not much money to be made hauling cargo in this thing." He gestures to his ship. "Tell you what, for 40%, I'll see to it that you arrive there safely. What do you think?"`
+			choice
+				`	"I was just thinking of doing the same, glad to know I'll have you with me!"`
```

Comment:
```suggestion
```

Reply:
Alternatively, add a new response following the suggested changes in line 5458.